snippets/acknowledge-the-update.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: SNIPPET
+[IMPORTANT]
+====
+Before you acknowledge the update, verify that there are no Kubernetes APIs in use that are removed in the version you are updating to.
+For example, in {product-title} 4.17, there are no API removals.
+See "Kubernetes API removals" for more information.
+====
+
+.Prerequisites
+
+* You have verified that APIs for all of the applications running on your cluster are compatible with the next Y-stream release of {product-title}.
+For more details about compatibility, see "Verifying cluster API versions between update versions".
```

Comment:
I might have missed it, but I don't see a link in the additional resources to "Verifying cluster API versions between update versions".

Reply:
Good catch, I added them now!